scripts/pager.gd

```diff
@@ -33,6 +37,14 @@ func next_error_code():
 		current_display += 1
 		current_display %= error_hexes.size()
 	set_display(current_display)
+	next_spr.play("click")
+
+func prev_error_code():
+	if(error_hexes.size() > 0 && current_display > 0):
+		current_display -= 1
+		current_display %= error_hexes.size()
```

Comment:
This will not cycle to the top of the error_hexes array. Removing the current_display > 0 conditional will work, I believe.

Reply:
I would have to test but I am pretty sure that it does cycle to the top. This is just saying that it can only go to a previous one as long as its not already at the top which should be 0.

I can test it when I am off work.